src/cpu/gemm_convolution_utils.cpp

```diff
@@ -1134,6 +1134,12 @@ status_t init_conf(conv_gemm_conf_t &jcp,
     const bool is_bwd_w = jcp.prop_kind == backward_weights;
     const bool is_fwd = !is_bwd_d && !is_bwd_w;
 
+    const auto dst_max_size
+            = static_cast<size_t>(jcp.iw) * jcp.ih * jcp.id * jcp.ic * 4;
+    const auto src_max_size
+            = static_cast<size_t>(jcp.ow) * jcp.oh * jcp.od * jcp.oc * 4;
+    if (dst_max_size > INT_MAX || src_max_size > INT_MAX) return unimplemented;
```

Comment:
Let's make it easier to figure why it returns unimplemented. Here and other places.

```suggestion
    VDISPATCH_CONV_IC(dst_max_size <= INT_MAX, VERBOSE_UNSUPPORTED_FEATURE,  "dst size > INT_MAX is not supported");
        VDISPATCH_CONV_IC(src_max_size <= INT_MAX, VERBOSE_UNSUPPORTED_FEATURE,  "src size > INT_MAX is not supported");
```

Reply:
done. Thanks!